test/integration/consts.go

```diff
@@ -5,23 +5,23 @@ package integration
 
 const (
 	admissionWebhookCert = `-----BEGIN CERTIFICATE-----
```

Comment:
Given that it's otherwise not possible to know at a glance what kind of certificate this is, it seems relevant to name it with "wildcard" in the name now that we've changed it to help convey:

```suggestion
	admissionWebhookWildcardCert = `-----BEGIN CERTIFICATE-----
```

Reply:
The need for understanding at a glance what kind of certificate this is makes sense.
The certificate now works for any service under the kong-system namespace (so it's no longer webhook specific). Therefore I propose a slightly more general approach in 0caed0e3:

```go
// kongSystemServiceCert is a testing TLS certificate with SAN *.kong-system.svc.
kongSystemServiceCert = `-----BEGIN CERTIFICATE----- ...`
```